src/styles/index.ts

```diff
@@ -3674,6 +3674,22 @@ const styles = (theme: ThemeColors) =>
             ...wordBreak.breakWord,
         },
 
+        quickActionTooltipWrapper: {
+            backgroundColor: colors.green100,
```

Comment:
cc @dubielzyk-expensify - are you thinking to use the same background color across themes? 

Either way, I think we should make this a variable that is stored in both themes, basically treating it like any other color style we use.

Reply:
If we go with the one specified above, then yes, I was gonna keep the same look regardless of theme as it feels good in both light and dark mode

Reply:
Okay sweet, thanks for confirming. Even though they are the same across both themes, I think we should still take the same approach we do for all other colors here.

Reply:
Also noting that we do have existing tooltip styles, so maybe we can consider this an extension of that... like `highlightTooltip` or something?

Reply:
I like that idea!